main/src/ca/uwaterloo/flix/api/lsp/provider/completion/CompletionUtils.scala

```diff
@@ -20,7 +20,8 @@ import ca.uwaterloo.flix.api.lsp.TextEdit
 import ca.uwaterloo.flix.api.lsp.provider.CompletionProvider.Priority
 import ca.uwaterloo.flix.language.ast.{Type, TypeConstructor, TypedAst}
 import ca.uwaterloo.flix.language.fmt.FormatType
-
+import ca.uwaterloo.flix.language.ast.Symbol
+import ca.uwaterloo.flix.language.ast.Symbol.ModuleSym
```

Comment:
Is this import dead? Usually we don't import `ModuleSym` directly, but always write `Symbol.ModuleSym`.